### ISSUE TITLE
FAILING_TEST_MATCHER doesn't match multi-line reasons

### ISSUE THREAD

**Issue by eapache** [supermarin/xcpretty#185]
We have a test suite which is failing with something like the following:

```
/Users/eapache/src/Tests.m:44: error: -[Tests testMyTest] : failed: caught "Error", "Err
Foo
Boo"
```

This results in the following output from xcpretty:

```
Tests
  testMyTest, failed: caught "Error", "Err
  /Users/eapache/srcTests.m:44
```

which is (in the original non-redacted case) fairly incoherent. Is there a way to make the regex multi-line and actually match the quotes so that we get the full error from xcpretty?

**Comment by tmyersjstar** [comment]
I think that the following regex might work:

`^\s*(.+:\d+):\serror:\s[\+\-]\[(.*)\s(.*)\]\s:(?:\s'.*'\s\[FAILED\],)?\s((?:(?!Test Case).*\n)*)`

With the exception of the error output having "Test Case" at the beginning of one of its additional lines.

**Comment by supermarin** [comment]
Will be addressed in 0.3
